app/src/main/contrib/actionTypes/CoverageTypes.ts

```diff
@@ -0,0 +1,33 @@
+import {CoverageSet} from "../../shared/models/Generated";
+
+export enum CoverageTypes {
+    COVERAGE_DATA_SETS_FETCHED = "COVERAGE_DATA_SETS_FETCHED",
+    COVERAGE_SET_FORMAT = "COVERAGE_SET_FORMAT",
+    COVERAGE_ONE_TIME_TOKEN_FETCHED = "COVERAGE_ONE_TIME_TOKEN_FETCHED",
+    COVERAGE_ONE_TIME_TOKEN_CLEAR = "COVERAGE_ONE_TIME_TOKEN_CLEAR"
+}
+
+export interface CoverageDataSetsFetched {
+    type: CoverageTypes.COVERAGE_DATA_SETS_FETCHED;
+    data: CoverageSet[];
+}
+
+export interface CoverageSetFormat {
+    type: CoverageTypes.COVERAGE_SET_FORMAT;
```

Comment:
A `CoverageSet` is a concept in our data model, so using `Set` as a verb here is a bit confusing! 

In general, if we're prefixing interface names in this way (and I agree it might be confusing to have 2 actions called `SetFormat`, one for demographics and one for coverage) I think the ideal way to do this is with namespaces:
https://www.typescriptlang.org/docs/handbook/namespaces.html

So this would become
```
export namespace Coverage {
    export interface DataSetsFetched {
        type: CoverageTypes.COVERAGE_DATA_SETS_FETCHED;
        data: CoverageSet[];
    }

    export interface SetFormat {
        type: CoverageTypes.COVERAGE_SET_FORMAT;
        data: string;
    }

    export interface OneTimeTokenFetched {
        type: CoverageTypes.COVERAGE_ONE_TIME_TOKEN_FETCHED;
        data: string;
    }

    export interface OneTimeTokenClear {
        type: CoverageTypes.COVERAGE_ONE_TIME_TOKEN_CLEAR;
    }
}

export type CoverageAction =
    | Coverage.DataSetsFetched
    | Coverage.SetFormat
    | Coverage.OneTimeTokenFetched
    | Coverage.OneTimeTokenClear
```

Then in `coverageActioncreators` you just need to import this namespace, and you can reference the actions individually in the same way (`Coverage.SetFormat` etc)

This seems like a sensible pattern for us to adopt in general, although particularly useful in the case of coverage and demographics; what do you think @MartinEden ?